packages/dara-components/js/common/stack/stack.tsx

```diff
@@ -16,7 +16,7 @@ import styled from '@darajs/styled-components';
 // The props are actually used deep within the useComponentStyles etc hooks
 /* eslint-disable react/no-unused-prop-types */
 interface StackProps extends LayoutComponentProps {
-    children: Array<ComponentInstance>;
+    children: Array<ComponentInstance | null>;
```

Comment:
Do we need to update this in more places for null? Not sure I understand why it was only updated on Stack

Reply:
I should actually revert it, I started implementing it here but moved it to base component instead.
DynamicComponent should make sure it's not actually null, good catch